sdk/translation/Azure.AI.Translation.Document/tests/ContainerDocumentTranslationClientTests.cs

```diff
@@ -47,5 +48,23 @@ public async Task StartSynchronousDocumentTranslation()
             Assert.IsNotNull(responseString);
             Assert.AreNotEqual(requestString, responseString);
         }
+
+        [Test]
+        public void DocumentTranslationSerializationTest()
```

Comment:
Can we add some comments it is for serialization sanity test, not for functional test

Reply:
Fixed